### PR TITLE
feat: toastify component

### PR DESCRIPTION
**Pull request by femacedogomes**
Theres a few changes to improve. The toastify is being activate when pages load, not when press the button.

### CODE REVIEW

File: src/components/atoms/Toastify/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  rtl,
+  buttonLabel,
+}) => {
+  const notify: any = () => {
+    toast(label);
+  };
+
+  return (
+    <div>
+      <button onClick={notify()}>{buttonLabel}</button>
```
Comment (renanpontez) [inline src/components/atoms/Toastify/index.tsx]: Remove this

File: src/components/atoms/Toastify/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        hideProgressBar={hideProgressBar}
+        newestOnTop={newestOnTop}
+        rtl={rtl}
+        theme={theme}
+      ></ToastContainer>
+    </div>
+  );
+};
+
+export default Toastify;
```
Comment (renanpontez) [inline src/components/atoms/Toastify/index.tsx]: Do we still need this file?

File: src/hooks/useTostify.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+//    }
+// })
+
+// useTostify({
+//   type: 'success',
+//   label: 'Hello World',
+//   options: {
+//     className: 'bg-scrummyOrange-500',
+//    }
+// })
```
Comment (renanpontez) [inline src/hooks/useTostify.tsx]: Remove this comments

File: src/pages/design-system.tsx [hunk 4]
```diff
@@ -14,12 +15,77 @@ import RadioButton from '../components/atoms/RadioButton';
...
+      label: 'Hello World',
+      type: 'warning',
+    });
+  };
+  const handleToastError = () => {
+    UseTostify({
+      label: 'Hello World',
+      type: 'error',
+    });
+  };
```
Comment (renanpontez) [inline src/pages/design-system.tsx]: change this to:

```suggestion
  const handleToast = (label, type) => UseTostify({ label, type });
```

Comment (renanpontez) [reply src/pages/design-system.tsx]: And in each button call: 
`onClick={() => handleToast('Hello', 'info')}`

**Comment by renanpontez** [comment]
Please check the comments and remember to always:

- Open the PR (when ready for review);
- Assign yourself;
- Look at the comments;

File: src/components/atoms/Toastify/index.tsx (hunk 2 above)
Comment (femacedogomes) [reply src/components/atoms/Toastify/index.tsx]: no more. thanks.

**Review by renanpontez — APPROVED** [review]
lgtm!

**Pull request merged**